qaas-app/src/entry-point.lisp

```diff
@@ -0,0 +1,226 @@
+;;;; qaas/src/entry-point.lisp
+;;;;
+;;;; Author: appleby
```

Comment:
I would love it if this file header could go away, or at least the "Author" part.  Makes sense when there is a single code-owner, not so much in a collaborative project.

Reply:
it is nice to have a point of contact, even if it goes stale